template/packages/frontend/package.json

```diff
@@ -23,12 +24,20 @@
     "react-dom": "16.13.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.12.3",
     "@hallettj/eslint-plugin-ts-graphql": "^0.1.1",
+    "@storybook/addon-docs": "^6.0.28",
+    "@storybook/react": "^6.0.28",
     "@types/node": "^14.14.6",
     "@types/react": "^16.9.55",
     "@types/react-dom": "^16.9.9",
+    "@types/storybook__react": "^5.2.1",
+    "babel-loader": "^8.2.0",
+    "babel-preset-react-app": "^10.0.0",
```

Comment:
I'm a bit bothered by adding these babel dependencies if we don't directly reference them. Are they necessary? This applies to `@babel/core` too.

Reply:
Seems that babel is required for storybook. 
https://github.com/storybookjs/storybook/issues/7156